internal/scaffold/role.go

```diff
@@ -140,6 +140,9 @@ func UpdateRoleForResource(r *Resource, absProjectPath string) error {
 		}
 		m := &map[string]interface{}{}
 		err = yaml.Unmarshal(d, m)
```

Comment:
Nit: Maybe also make this the one-liner assignment and error check to match line 98?

 `if err := yaml.Unmarshal(d, m); err != nil {`